### DISCUSSION TITLE
What's the mean of CN column？

### DISCUSSION THREAD

**Discussion by yuanpingdu** [etal/cnvkit#995] [General]
I have run CNVKit to analysis diploid, and the CN column result is：
cn
0
1
2
3
4
5
6
7
8
9
10
11
12
14
20
57
81
What‘s the mean of cn=0? cn=1? ... cn=81?
How to kown the dup result and deletion result?

**Comment by etal** [comment]
Integer copy number, converted from segment log2 ratios and adjusted for user-specific purity and ploidy by the `call` command. In a diploid genome, 0 = full deletion, 1 = hemizygous (partial or single-copy deletion), 2 = copy number neutral, 3 = single copy gain (dup), 4+ = multi-copy gain/amplification.
